sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/ConfigurationAsyncClient.java

```diff
@@ -125,8 +126,9 @@
      * {@codesnippet com.azure.data.appconfiguration.configurationasyncclient.addSettingWithResponse#ConfigurationSetting}
      *
      * @param setting The setting to add to the configuration service.
-     * @return A REST response containing the {@link ConfigurationSetting} that was created, if a key collision occurs or the key
-     * is an invalid value (which will also throw HttpResponseException described below).
+     * @return A REST response containing the {@link ConfigurationSetting} that was created, if a key collision occurs
+     *     or the key
+     *     is an invalid value (which will also throw HttpResponseException described below).
```

Comment:
Is this line break where you meant it to be? The previous line has less than 120 characters.